src/styles/_catalog.scss

```
@@ -1,22 +1,29 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
```

you don't need `repeat(1, ...)`
```suggestion
  grid-template-columns: 200px;
```